<!DOCTYPE HTML>
<!--
	Twenty by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Projects - Anthony Piot</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<style>
		.text2 {
			padding: 20px;
		}
		#header.alt2 {
			background: transparent;
			box-shadow: none;
			color: #7c8081;
			padding: 2em 2.5em;
			top: 1em;
		}
	</style>
	<body class="no-sidebar is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<header id="header" class="alt2">
					<nav id="nav">
						<ul>
							<li class="current"><a href="index.html">Home</a></li>
							<li class="submenu">
								<a href="#">Menu</a>
								<ul>
									<li><a href="about_me.html">About me</a></li>
									<li><a href="research.html">Research</a></li>
									<li><a href="contact.html">Contact</a></li>
								</ul>
							</li>
						</ul>
					</nav>
				</header>

			<!-- Main -->
				<article id="main">

					<header class="special container">
						<span class="icon solid fa-search"></span>
					</header>

						<!-- Summary -->
						<section class="wrapper style4 container">

							<!-- Content -->
								<div class="content">
									<section>
										<p>My research focuses on answering evolutionary questions using genomic data. I worked on a variety of approach such as
											population genomics, quantitative genetics, phylogenetics and comparative genomics. 
										<br>
										<br>The experience gained in these different topics led me to develop a more specific interest in how heritable 
											biological information interacts with an organism's environment to produce phenotypic variation and how such variation 
											drives evolution.
										</p>
									</section>
								</div>
						</section>

						<!-- Current projects -->
						<div class="text">
							<header>
								<h2><center>Current projects</center></h2>
							</header>
						</div>

						<section class="wrapper style4 container">
							<div class="text">
									<header>
										<h3><center>Lyme disease</center>
											<br></br>
										</h3>
									</header>
							</div>	
							<div class="container2">
								<div class="text2">
									<p>Lyme disease is transmitted by a tick borne bacteria of the Borrelia genus.
										Bacteria of the Borrelia genus have some but very complex genomes constitued of a linear 
										chromosome and many linear and circular plasmids.
										Using long read sequencing, I am reconstructing genomes of different Borrelia strains in order to learn more about the 
										geographic spread and genome evolution of this human pathogen.</p>
								</div>
								<div class="image">
									<img src="images/tick.jpg" alt="" width ="500" height="375"/>
								</div>
							</div>	
						</section>

						<section class="wrapper style4 container">
							<div class="text">
									<header>
										<h3><center>Poplar</center>
											<br></br>
										</h3>
									</header>
							</div>
							<div class="container2">
								<div class="image">
									<img src="images/poplar.jpg" alt="" width ="500" height="375"/>
								</div>
								<div class="text2">
									<p>Wood is widely used as a renewable material and energy source thanks to its physical and chemical properties.
										Tailoring wood properties for human usage is therefore important for many sectors. Using whole genome sequencing, 
										I searched for large effect genetic variants that could rapidly be used in molecular breeding to modify wood properties.
										I focused on disruptive variants, defined as small genetic variants changing gene coding sequences and resulting 
										in a deleterious effect on gene products.</p>
								</div>
							</div>	
								</div>
							</div>
						</section>

						<!-- Past projects -->
						<div class="text">
							<header>
								<h2><center>Past projects</center></h2>
							</header>
						</div>
						
						<section class="wrapper style4 container">
							<div class="text">
									<header>
										<h3><center>GENOSCAN</center>
											<br></br>
										</h3>
									</header>
							</div>	
							<div class="container2">
								<div class="text2">
									<p> As part of the <a href="https://www.sentinelnorth.ulaval.ca/en/research/ecogenomics-mining-areas-sustainable-canadian-north-genoscan">
										<span class="label">GENOSCAN</span></a>	project, I reconstructed the population structure of black spruce individuals 
										growing inside and around a mining area in northern Qu√©bec. After sampling biological material from spruce individuals, we
										used Genotyping-By-Sequencing (GBS) data to identify genome-wide variants across the studied individuals and 
										reconstruct their underying population structure.  
									</p>
								</div>
								<div class="image">
									<img src="images/black_spruce.jpg" alt="" width ="300" height="400"/>
								</div>
							</div>	
						</section>

						<section class="wrapper style4 container">
							<div class="text">
									<header>
										<h3><center>C4 photosynthesis</center>
											<br></br>
										</h3>
									</header>
							</div>
							<div class="container2">
								<div class="image">
									<img src="images/grass.jpg" alt="" width ="500" height="375"/>
								</div>
								<div class="text2">
									<p>C4 photosynthesis evolved multiple times in the grass family (Poaceae) from the ancestral C3 photosynthesis state, 
										reprensenting an amazing case of parallel evolution.
										Using more than a hundred chloroplast genomes from numerous grass species and replacing them in a phylogenetic context 
										we identified one gene (<i>rbcL</i>) under strong positive selection following the C3-C4 transition, while one third of 
										plastid genes were found to be have evolved under positive selection in across all the studied species. 
									</p>
								</div>
							</div>	
								</div>
							</div>
						</section>
						
						<!-- Links -->
						<section class="wrapper style1 container special">
							<div class="row">
								<div class="col-7 col-12-narrower">

									<section>
										<footer>
											<ul class="buttons">
												<li><a href="about_me.html" class="button small">About me</a></li>
											</ul>
										</footer>
									</section>

								</div>
								<div class="col-3 col-12-narrower">

									<section>
										<footer>
											<ul class="buttons">
												<li><a href="contact.html" class="button small">Contact</a></li>
											</ul>
										</footer>
									</section>

								</div>
							</div>
						</section>

				</article>

			<!-- Footer -->
			<footer id="footer">
				<ul class="copyright">
					<li>&copy; </li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
				</ul>

			</footer>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollgress.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>